cg_hermes/config/raredisease.py

```diff
@@ -0,0 +1,147 @@
+
+
+RAREDISEASE_COMMON_TAGS = {
+    frozenset(["chromograph_upd", "sites"]): {
+        "tags": ["chromograph", "upd", "sites"],
+        "is_mandatory": False,
+        "used_by": ["scout"],
+    },
+    frozenset(["chromograph_upd", "regions"]): {
+        "tags": ["chromograph", "upd", "regions"],
+        "is_mandatory": False,
+        "used_by": ["scout"],
+    },
+    frozenset(["chromograph_cov", "tcov"]): {
+        "tags": ["chromograph", "tcov"],
+        "is_mandatory": False,
+        "used_by": ["scout"],
+    },
+    frozenset(["chromograph_rhoviz", "autozyg"]): {
+        "tags": ["chromograph", "autozyg"],
+        "is_mandatory": False,
+        "used_by": ["scout"],
+    },
+    frozenset(["rank_and_filter", "clinical"]): {
+        "tags": ["vcf-snv-clinical"],
+        "index_tags": ["vcf-snv-clinical-index"],
+        "is_mandatory": True,
+        "bundle_id": True,
+        "used_by": ["scout"],
+    },
+    frozenset(["rank_and_filter", "research"]): {
+        "tags": ["vcf-snv-research"],
+        "index_tags": ["vcf-snv-research-index"],
+        "is_mandatory": True,
+        "used_by": ["scout"],
+    },
+    frozenset(["expansionhunter", "sv_str"]): {
+        "tags": ["vcf-str"],
+        "index_tags": ["vcf-str-index"],
+        "is_mandatory": False,
+        "used_by": ["scout"],
+    },
+    frozenset(["expansionhunter", "str_variants"]): {
+        "tags": ["expansionhunter", "vcf-str"],
+        "is_mandatory": False,
+        "used_by": ["scout"],
+    },
+    frozenset(["expansionhunter", "str_alignment"]): {
+        "tags": ["expansionhunter", "bam"],
+        "index_tags": ["expansionhunter", "bam-index"],
+        "is_mandatory": False,
+        "used_by": ["scout"],
+    },
+    frozenset(["expansionhunter", "variant_catalog"]): {
+        "tags": ["expansionhunter", "variant-catalog"],
+        "is_mandatory": False,
+        "used_by": ["scout"],
+    },
+    frozenset(["alignment"]): {
+        "tags": ["cram"],
+        "index_tags": ["cram-index"],
+        "is_mandatory": False,
+        "used_by": ["scout"],
+    },
+    frozenset(["gens_generatedata", "baf"]): {
+        "tags": ["gens", "fracsnp", "bed"],
+        "index_tags": ["gens", "fracsnp", "bed-index"],
+        "is_mandatory": False,
+        "used_by": ["scout"],
+    },
+    frozenset(["gens_generatedata", "cov"]): {
+        "tags": ["gens", "coverage", "bed"],
+        "index_tags": ["gens", "coverage", "bed-index"],
+        "is_mandatory": False,
+        "used_by": ["scout"],
+    },
+    frozenset(["pedigree", "pedigree_fam"]): {
+        "tags": ["pedigree"],
+        "is_mandatory": True,
+        "used_by": ["scout"],
+    },
+    frozenset(["multiqc", "html"]): {
+        "tags": ["multiqc-html"],
+        "is_mandatory": True,
+        "used_by": ["scout"],
+    },
+    frozenset(["multiqc", "json"]): {
+        "tags": ["multiqc-json"],
+        "is_mandatory": True,
+        "used_by": ["storage"],
```

Comment:
`storage` is a deprecated tag.

Use "long-term-storage" for keeping the files in HK for more than a year, and "clinical-delivery" for files to be delivered to Caesar